dev-docs/modules/iabCatalogTranslation.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: page_v2
+page_type: module
+title: Module - IAB Category Translation
+description: Converts between ad agency brand categories and IAB brand categories.
+module_code : CategoryTranslation
+display_name : CategoryTranslation
+enable_download : true
+sidebarType : 1
+---
+
+# IAB Category Translation
+
+{:.no_toc}
+
+This module converts the IAB sub category to Freewheel industry group identifiers. The Freewheel identifiers ensure competitve separation of industries and products. 
+
+Each bid request must return one [IAB subcategory](https://support.aerserv.com/hc/en-us/articles/207148516-List-of-IAB-Categories).
+
+The module provides the following: 
+
+- Converts IAB subcategories to a Freewheel industry group identifier.
+
+## How to use the module:
+
+1. A Prebid.js package is built that contains this module and the [Freewheel]({site.github.url}}/dev-docs/module/freewheel.html) module.  
+2. The inclusion of this module causes Prebid to download a mapping file to local storage. The user also has the option to provide their own mapping file. 
+3. At runtime, brand category translation happens as needed. 
+
+
+## Using A Custom Map File
+The IAB Category Translation module uses a default mapping file to convert adserver categories to IAB sub categories. If a publisher prefers to use their own mapping file they will need to set the URL location of that file. They can do so by adding the following to their Prebid.js configuration: 
+
+```
+pbjs.setConfig({
+    "brandCategoryTranslation": {
+       "translationFile": "<url_to_file>"
+    }
+});
+```
+
+This file will be stored locally to expedite the conversion process. If a publisher opts to not provide a conversion mapping file Prebid will use its default conversion mapping file. 
+
+Publishers should ensure that the JSON returned from their custom file is valid for Prebid by adhering to the following structure: 
+
+{
```

Comment:
when previewing, this code block isn't showing as formatted.

Reply:
Fixed.